docs/maui/essentials/folder-picker.md

```diff
@@ -0,0 +1,70 @@
+---
+title: FolderPicker - .NET MAUI Community Toolkit
+author: VladislavAntonyuk
+description: The FolderPicker provides the ability to pick a folder from the file system.
+ms.date: 12/11/2022
+---
+
+# FolderPicker
+
+The `FolderPicker` provides the ability to pick a folder from the file system.
+
+The following preconditions required for the `FolderPicker`:
+# [Android](#tab/android)
+
+Add permissions to `AndroidManifest.xml`:
+
+```xml
+<uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
+```
+
+# [Tizen](#tab/tizen)
+
+Add permissions to `tizen-manifest.xml`:
+
+```xml
+<privilege>http://tizen.org/privilege/mediastorage</privilege>
+<privilege>http://tizen.org/privilege/externalstorage</privilege>
+```
+
+## Syntax
+
+### C#
+
+The `FolderPicker` can be used as follows in C#:
+
+```csharp
+async Task PickFolder(CancellationToken cancellationToken)
+{
+    try
+    {
+        var folder = await FolderPicker.Default.PickAsync(cancellationToken);
+        await Toast.Make($"Folder picked: Name - {folder.Name}, Path - {folder.Path}", ToastDuration.Long).Show(cancellationToken);
+    }
+    catch (Exception ex)
+    {
+        await Toast.Make($"Folder is not picked, {ex.Message}").Show(cancellationToken);
+    }
+}
+```
+
+### Folder
+
+The `Folder` record represents a folder in the file system. It defines the following properties:
+
+- Path contains a Folder path.
+- Name contains a Folder name.
+
+## Methods
+
+|Method  |Description  |
+|---------|---------|
+| PickAsync | Asks for permission and allows selecting a folder in the file system. |
+
```

Comment:
Do we want to include details on how to register these essentials with the `MauiAppBuilder`?

Reply:
It doesn't require registration and can be used as static property Default.

Reply:
I know it doesn't require registration, I was just thinking from a best practices point of view, much like we do with the sample app

Reply:
Thank you, fixed